src/Nethermind/Nethermind.Runner/configs/energyweb.json

```diff
@@ -12,10 +12,9 @@
   },
   "Sync": {
     "FastSync": true,
-    "SnapSync": true,
```

Comment:
@kamilchodola do you know why this is being removed here?

Reply:
Nice catch! Will see why is that happening - before then please don't merge this